infrastructure/main.tf

```diff
@@ -7,7 +12,7 @@ module "pdf-service-api" {
   subscription = "${var.subscription}"
 
   app_settings = {
-    S2S_URL = "${var.s2s_url}"
+    S2S_URL = "${local.s2s_url}"
```

Comment:
can we try it in aat first? maybe not go to prod on the first attempt

Reply:
Of course, I'm trying it out on saat now